Create ui-c8y-1023-22-1-fixed-untranslated-texts-with-guide-links-10535-graft-release-cd-10798.md (#4107)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/ui-c8y-1023-22-1-fixed-untranslated-texts-with-guide-links-10535-graft-release-cd-10798.md

---
date: ""

> title: Fixed translation issues with texts containing links to documentation

Reviewer: ```suggestion
title: Fixed localization issues with texts containing links to the documentation
```

product_area: Application enablement & solutions
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-YbYJ3gLU_
    label: Web SDK
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-65540
version: 1023.22.1
---

> This change fixes an issue with broken translation of empty state messages with documentation links in several places.

Reviewer: ```suggestion
This change fixes a localization issue that occurred when empty state messages included documentation links. These messages were not displaying properly in multiple places throughout the application.
```

> This change fixes a localization issue that occurred when empty state messages included documentation links. These messages were not displaying properly in multiple places throughout the application.

Reviewer 2: ```suggestion
This change fixes a localization issue that occurred when empty state messages included documentation links. These messages were not displayed properly in multiple places throughout Administration and Cockpit.
```

> This change fixes a localization issue that occurred when empty state messages included documentation links. These messages were not displayed properly in multiple places throughout Administration and Cockpit.

Reviewer: ```suggestion
This change fixes a localization issue that occurred when empty state messages included documentation links. These messages were not displayed properly in multiple places throughout the Administration and Cockpit applications.
```

---
date: ""

> title: Fixed untranslated texts with guide links (#10535) [GRAFT][release/cd] (#10798)

Reviewer 2: ```suggestion
title: Fixed translation issues with texts containing links to documentation
```

product_area: Application enablement & solutions
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-YbYJ3gLU_
    label: Web SDK
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-65540
version: 1023.22.1
---
[MTM-65540] Added unit test for the dynamic guide link case, added `ngNonBindable` where it was missing.

**Affected places:**

Administration:

- Tenants > Subtenants > [tenant] > Custom properties (when no tenant
properties defined in Properties library)

Cockpit:

- [dashboard] > Add widget > Data points graph > [preview when no data
point selected]
- [dashboard] > Add widget > Info gauge > [preview when no data point
selected]
- [dashboard] > Add widget > Linear gauge > [preview when no data point
selected]
- [dashboard] > Add widget > KPI widget > [preview when no data point
selected]
- [dashboard] > Add widget > Data points table > [preview when no data
point selected]
- [dashboard] > Add widget > Data points table > [preview when selected

> data point has no measurements]

Reviewer 2: ```suggestion
This change fixes an issue with broken translation of empty state messages with documentation links in several places.
```
